app/templates/serviceLearning/slcQuestionnaire.html

```diff
@@ -0,0 +1,46 @@
+{% extends "base.html" %}
+{% block scripts %}
+
+  {{super()}}
+  <style media="screen"></style>

```

Comment:
You're including a style comment in the scripts block. There's another block for that, right?